expyfun/stimuli/tests/test_tracker.py

```diff
@@ -147,15 +147,14 @@ def test_tracker_dealer():
     # test TrackerDealer with TrackerUD
     trackers = [[TrackerUD(None, 1, 1, 0.06, 0.02, 20, 'reversals', 1)
                 for _ in range(2)] for _ in range(3)]
-    dealer_ud = TrackerDealer(trackers)
-    dealer_ud = TrackerDealer(np.array(trackers))
+    dealer_ud = TrackerDealer(callback, trackers)
+    dealer_ud = TrackerDealer(callback, np.array(trackers))
 
     # can't respond before you pick a tracker and get a trial
     assert_raises(RuntimeError, dealer_ud.respond, True)
     rand = np.random.RandomState(0)
-    dealer_ud.get_trial()
-    while not dealer_ud.stopped:
-        sub, x_current = dealer_ud.get_trial()
+    
+    for sub, x_current in dealer_ud:
         dealer_ud.respond(rand.rand() < x_current)
         assert(np.abs(dealer_ud[0, 0].n_reversals -
                       dealer_ud[0, 0].n_reversals) <= 1)
```

Comment:
Heh, I think that should be comparing the number of reversals from two *different* trackers within the dealer, to make sure they never get off by more than 1. But it clearly doesn't do that does it?